js/phemex.js

```diff
@@ -2932,6 +2933,33 @@ module.exports = class phemex extends Exchange {
         };
     }
 
+    async setMarginMode (marginType, symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' setMarginMode() requires a symbol argument');
+        }
+        marginType = marginType.toLowerCase ();
+        if (marginType !== 'isolated' && marginType !== 'cross') {
+            throw new BadRequest (this.id + ' setMarginMode() marginType argument should be isolated or cross');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        if (market['type'] !== 'swap') {
+            throw new BadSymbol (this.id + ' setMarginMode() supports swap contracts only');
+        }
+        let leverage = this.safeInteger (params, 'leverage');
+        if (marginType === 'cross') {
+            leverage = 0;
```

Comment:
Is there a reason for making the default value leverage equal to zero on cross margin?

Reply:
The only way to set the margin mode to cross is by setting leverage to 0 for this exchange